gcloud-java-resourcemanager/src/main/java/com/google/gcloud/spi/DefaultResourceManagerRpc.java

```diff
@@ -82,7 +88,7 @@ public Project get(String projectId, Map<Option, ?> options) throws ResourceMana
           .execute();
     } catch (IOException ex) {
       ResourceManagerException translated = translate(ex);
-      if (translated.code() == HTTP_FORBIDDEN) {
+      if (translated.code() == HTTP_FORBIDDEN || translated.code() == HTTP_NOT_FOUND) {
```

Comment:
Just out of curiosity and related to #440: is 403 returned for both the not found and permission denied case? Is this done not to disclose info on existing projects?